src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -27,7 +27,7 @@ public static DirectoryInfo GetParent(String path)
                 throw new ArgumentException(SR.Argument_PathEmpty, "path");
             Contract.EndContractBlock();
 
-            String fullPath = PathHelpers.GetFullPathInternal(path);
+            String fullPath = Path.GetFullPath(path);
```

Comment:
Excellent; I love that we're switching to use the public API surface area rather than an internal helper.